__tests__/e2e/tokens.spec.ts

```diff
@@ -11,12 +11,112 @@ describe('Tokens', () => {
     expect(response.body).toMatchObject({
       id: '24f4f5f7-c29e-4707-961a-3515be5a2f3e',
       capture_id: 'f6c0e710-d80a-4e93-a9d2-d4edb52856af',
-      wallet_id: 'eecdf253-05b6-419a-8425-416a3e5fc9a0',
+      wallet_id: '01c75527-d9c3-437b-bf4b-167c60849e23',
       transfer_pending: false,
       transfer_pending_id: null,
       // created_at: '2021-02-18T23:53:29.172Z',
       // updated_at: '2021-02-18T23:53:29.172Z',
       claim: false,
     });
   });
+
+  it('/tokens?wallet=bd60973b-2f08-45c5-afb3-7ec018180f17', async () => {
+    const response = await supertest(app).get(
+      '/tokens?wallet=bd60973b-2f08-45c5-afb3-7ec018180f17',
+    );
+
+    expect(response.status).toBe(200);
+    expect(response.body.total).toBe(10);
+    expect(response.body).toMatchObject({
+      tokens: expect.anything(),
+    });
+
+    expect(response.body.tokens[0]).toMatchObject({
+      wallet_id: 'bd60973b-2f08-45c5-afb3-7ec018180f17',
+    });
+
+    expect(response.body.tokens[4]).toMatchObject({
+      wallet_id: 'bd60973b-2f08-45c5-afb3-7ec018180f17',
+    });
+
+    expect(response.body.tokens[9]).toMatchObject({
+      wallet_id: 'bd60973b-2f08-45c5-afb3-7ec018180f17',
+    });
+  });
+
+  it('/tokens?wallet=Malinda51', async () => {
+    const response = await supertest(app).get('/tokens?wallet=Malinda51');
+
+    expect(response.status).toBe(200);
+    expect(response.body.total).toBe(12);
+    expect(response.body).toMatchObject({
+      tokens: expect.anything(),
+    });
+
+    expect(response.body.tokens[0]).toMatchObject({
+      wallet_id: 'eecdf253-05b6-419a-8425-416a3e5fc9a0',
+    });
+
+    expect(response.body.tokens[7]).toMatchObject({
+      wallet_id: 'eecdf253-05b6-419a-8425-416a3e5fc9a0',
+    });
+
+    expect(response.body.tokens[11]).toMatchObject({
+      wallet_id: 'eecdf253-05b6-419a-8425-416a3e5fc9a0',
+    });
+  });
+
+  it('/tokens?wallet=Dave.Mertz68&withPlanter=true&withCapture=false', async () => {
+    const response = await supertest(app).get(
+      '/tokens?wallet=Dave.Mertz68&withPlanter=true&withCapture=false',
+    );
+
+    expect(response.status).toBe(200);
+    expect(response.body.total).toBe(10);
+    expect(response.body).toMatchObject({
+      tokens: expect.anything(),
+    });
+
+    expect(response.body.tokens[0]).toMatchObject({
```

Comment:
Here how are you making sure the `capture _id` is not in the returned object? I believe `toMatchObject` does not check for key-value pairs that are not supposed to be in the object. By that I mean, I can still have `capture_id` in my returned object and still pass the test. Correct me if I am wrong.

Reply:
@dagmawig you are absolutely spot on.
But I think the capture_id will be there and  capture_photo_url won't be there.
@dadiorchen am I, right?

Reply:
It depends on what `withCapture` variable is implying. Whichever it's indicating make sure it doesn't include that variable when set to `false` to make sure your query works.